src/AbstractRepository.php

```diff
@@ -52,4 +54,29 @@ protected function sanitizePath($path)
 
         return Path::canonicalize($path);
     }
+
+    /**
+     * @param string $path
+     *
+     * @return PuliResource
+     */
+    protected function followLinksOrFail($path)
+    {
+        $resolvedWithLinks = $this->followLinks($path);
```

Comment:
The `followLinks` method fully resolves the links so it will return the final target. I'll add this in the phpdoc.